unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WelcomeHUD/WelcomeHUDView.cs

```diff
@@ -29,10 +29,7 @@ public static WelcomeHUDView CreateView(bool hasWallet)
 
     public void Initialize(UnityAction OnConfirm, UnityAction OnClose)
     {
-        confirmButton.onClick.RemoveAllListeners();
-        confirmButton.onClick.AddListener(OnConfirm);
-
-        closeButton.onClick.RemoveAllListeners();
-        closeButton.onClick.AddListener(OnClose);
+        confirmButton.onPointerDown += OnConfirm;
+        closeButton.onPointerDown += OnClose;
```

Comment:
The same as before, maybe we should unsubscribe these subscriptions somewhere.

Reply:
done!